dev/io.openliberty.microprofile.faulttolerance.4.1.telemetry.metrics.2.0/.classpath

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
```

Comment:
Check Java version vs. Java requirements for the feature.

Reply:
Setting this to 11 to match the telemetry bundles.

Reply:
I think this makes sense but:

1. You need to also add this to the bnd.bnd:
    ```
    javac.source: 11
    javac.target: 11
    ```
2. Make sure the change is in whichever of the two autofeature bundles you decide to keep.